guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: guide
+title: Bitcoin backups
+description: A beginner's guide to Bitcoin backups.
+nav_order: 11
+parent: Foundations
+permalink: /guide/foundations/backups/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/foundations/wallet-interoperability/wallet-interoperability-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+A guide meant to lower the barrier for first time users of self-custody wallets.
+
+Illustration sources
+
+// 
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/foundations/backups/backups.jpg"
+   retina = "/assets/images/guide/foundations/backups/backups@2x.jpg"
+   mobile = "/assets/images/guide/foundations/backups/backups-mobile.jpg"
+   mobileRetina = "/assets/images/guide/foundations/backups/backups-mobile@2x.jpg"
+   alt-text = "Several devices exchanging data"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Bitcoin backups
+
+The self-custody of bitcoin is an important right, but also a financial responsibility that can take practice getting used to, and scale from pocket money to life savings.
+
+Making a secure backup of your wallet should be straightforward. This guide is meant to provide a starting point for simple bitcoin backups. From a basic setup, you can increase the security level as your funds grow. 
+
+
+## What are backups?
+
+When we talk about backups, we mean saving the information needed to restore access to a wallet outside of the application that created the private keys. Most often, this will be the *recovery-phrase* of of 12 or 24 words, but can also include other information like a *pass-phrase*.
+
+The main risks that backups need to protect against are:
+
+- Loss of wallet - you lost the device and application holding the private key
+- Remote theft - your private key got discovered online by a bad actor
+- Physical damage - your backup got destroyed
+- Theft on discovery - your backup got discovered in person by a bad actor
+- Catastrophic events - your backup location got destroyed
+- Inheritance loss - next-of-kin were unable to access your wallet
+
+
+## Backup levels, basic to advanced
+
+A basic backup can be done in a few minutes, while more advanced setups can require the purchase of dedicated material and more effort. 
+
+Let’s look at some real life examples of backup methods for single-key wallets and what they protect against. (Multi-key setups require their own writeup.)
```

Comment:
Would be good to link to appropriate sections in PKM that cover single-key wallets and multy-key ones. Also we should double-check if that's how we call this setups in PKM, for consistency.

Reply:
It's not quite like for like, as that chapter distinguishes between Personal and Shared wallets (which can both have single or multi setups). So for this purpose I think it could be more confusing than helpful to link.